azurelinuxagent/ga/remoteaccess.py

```diff
@@ -102,60 +102,78 @@ def run(self):
                       message=msg)
 
     def handle_remote_access(self):
+        # Get JIT user accounts.
+        all_users = self.os_util.get_users()
+        existing_jit_users = [u[0] for u in all_users if self.validate_jit_user(u[4])]
```

Comment:
This should be a set instead of a list. You check to see if names are in `existing_jit_users` in multiple places, and the one time you iterate the collection, the order doesn't appear to be relevant.